library/packages/src/lib/y2packager/product_upgrade.rb

```diff
@@ -44,6 +44,17 @@ class ProductUpgrade
       ["openSUSE"]                                                        => "SLES"
     }.freeze
 
+    # This maps the installed products to removed products.
```

Comment:
maybe make it even more clear like `This maps has as keys list of installed products and as value what products will be removed. All products in key have to be installed.`